src/electron/views/automated-checks/components/maximize-restore-button.tsx

```diff
@@ -0,0 +1,24 @@
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import * as React from 'react';
+
+import { RestoreIcon } from 'common/icons/restore-icon';
+import { NamedFC } from 'common/react/named-fc';
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export interface MaximizeRestoreButtonProps {
+    isMaximized: boolean;
+    onClick: () => void;
+}
+
+export const MaximizeRestoreButton = NamedFC<MaximizeRestoreButtonProps>('TitleBar', (props: MaximizeRestoreButtonProps) => {
+    const content = props.isMaximized ? null : <RestoreIcon />;
+    const icon = props.isMaximized ? { iconName: 'Stop' } : null;
```

Comment:
It looks like we refer to the icons in camelCase (eg, `'stop'`) in most other places - should probably be consistent (not sure if there's any linux-specific bugs lurking here)

Reply:
We seem to [usually](https://github.com/microsoft/accessibility-insights-web/blob/ccdbcf4a141364dba315e6c56495ff2211f925e9/src/popup/components/header-contextual-menu.tsx#L37) [refer](https://github.com/microsoft/accessibility-insights-web/blob/ccdbcf4a141364dba315e6c56495ff2211f925e9/src/popup/launch-pad-row-configuration-factory.ts#L30) [to](https://github.com/microsoft/accessibility-insights-web/blob/ccdbcf4a141364dba315e6c56495ff2211f925e9/src/DetailsView/components/start-over-dropdown.tsx#L44) single word iconNames capitalized, so I have a difference instinct on this one...I don't particularly care one way or the other, but this seems closer to consistent in my opinion